jaeger-core/build.gradle

```diff
@@ -19,6 +20,18 @@ dependencies {
     signature 'org.codehaus.mojo.signature:java16:1.1@signature'
 }
 
+apply plugin: 'com.github.johnrengelman.shadow'
```

Comment:
I don't think we should use shadow. Is there even request for it? 

cc @jpkrohling